control/control.go

```diff
@@ -8,13 +8,25 @@ import (
 	"os"
 	"time"
 
-	"github.com/joyent/containerpilot/events"
+	"github.com/prometheus/client_golang/prometheus"
 	log "github.com/sirupsen/logrus"
+
+	"github.com/joyent/containerpilot/events"
 )
 
 // SocketType is the default listener type
 var SocketType = "unix"
 
+var collector *prometheus.CounterVec
+
+func init() {
+	collector = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
I think I see what you mean now. In theory you could define all of your collectors or even just your prometheus option structs in a separate file and import them into the locations where you setup the collectors. I'm guessing thats the idea you mentioned in Slack but then went with placing each collector across the app.

On the opposite side, it's three definitions across this PR, one of which covers 80% of the events. LGTM.